source/standards/accounts-with-third-parties.html.md.erb

```diff
@@ -1,143 +1,139 @@
 
-When developing a service, your team will often require the use of
-third-party services.  To access these services, you will need some
-sort of user account.  This page provides guidance on how to manage
-those accounts.
-
-## Terms
-
-In this page we distinguish between an account and an individual:
-
-- An *account* means an account with an identifier such as a username or
-  email address
-- An *individual* means a human being
-- A set of *credentials* means some sort of secret used to gain access
-  to a user account (for example: a password, a secret access key)
-
-## Goals and principles
-
-When managing credentials we have a few goals:
-
-- Ensure that the right people have access
-- Ensure that the wrong people do not have access
-  - Particularly: ensure when people leave the organization, their
-    access is revoked or expired
-- Identify who did what and when (audit)
-  - This goal has varying importance depending on context
-  - Technical limitations may prevent the goal from being achieved, or
-    only provide partial audit
-
-To meet these goals, we have some principles:
-
-- Prefer to minimise the number of secrets that an individual needs to
-  keep
-  - This helps with revocation because there are fewer sets of
-    credentials to revoke when someone leaves
-  - Passwords are particularly problematic secrets. Other forms of
-    secrets (eg secret access keys or OAuth tokens) are more easily
-    rotated.
-- No individual should ever have to use credentials associated with
-  another individual
-  - This would be counter to the audit goal
-- There should be no need to share credentials between teams
-  - Again, this is problematic for audit
-  - This is also problematic for knowing when credentials should be
-    revoked
-  - Technical limitations may mean this is unavoidable sometimes
+When developing a service at GDS, your team will often need to use
+third-party services. To access these services, you will need some
+sort of user account, this guidance explains how to manage
+such accounts.
+
+## Distinguishing between user accounts, individuals and credentials
+
+In this guidance we distinguish between accounts, individuals and credentials by saying:
+
+- an *account*  has an identifier such as a username or an email address
+- an *individual* refers to a human being
+- a set of *credentials* means a type of secret used to gain access
+  to a user account, for example a password or secret access key
+
+## Account management goals and principles
+
+When managing account credentials there are certain goals and guiding principles we recommend you use.
+
+### Account management goals
+
+You should manage access to your service by:
+
+- ensuring the right people have access
+- ensuring the wrong people do not have access
+
+You must ensure that when people leave GDS, their access is revoked or expired. You should also identify who did what and when (audit), this has varying importance depending on its context.
```

Comment:
I don't think that "You should also identify who did what and when" is correct here.  Some teams care much more about audit than others.

Reply:
in that case, if we're not mandating this how about "we recommend you identify who did what and when" to soften the tone?

Reply:
again, I think it's up to teams to decide what their risk appetite is and therefore what actions to take.  We're coming from the position of "you might want to consider audit. if you do, here's what to think about" rather than "we think you should do audit"